examples/ruby/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+function announce() {
+  echo "++++++++++"
+  echo "$@"
+  echo "++++++++++"
+}
+
+if [[ ! -d "./out/ruby" ]]; then
+  announce "Client not found. Generating..."
+  ./bin/generate_client ruby > /dev/null
+  echo
+fi
+
+if [[ -z "$(docker-compose ps -q)" ]]; then
+  announce "Environment not found. Spinning up..."
+  ./bin/start_conjur > /dev/null
+  echo
+fi
+
+announce "Building Ruby API client gem..."
+gem build ./out/ruby/openapi_client.gemspec
+echo
+
+announce "Installing Ruby API client gem..."
+sudo gem install ./out/ruby/openapi_client-1.0.0.gem
+echo
```

Comment:
For initial implementation this is maybe fine but we really don't want to install this gem system-wide on the user's machine. Can we do this in a container or `rvm`/`rbenv`?